cfn/cfn.go

```diff
@@ -76,13 +76,17 @@ func Start(h Handler) {
 
 	// MODE is an environment variable that is set ONLY
 	// when contract test are performed.
-	if mode, ok := os.LookupEnv("MODE"); ok == true {
+	mode, ok := os.LookupEnv("MODE")
+
+	switch ok {
```

Comment:
There are only two outcomes here so we can simplify this. Don't need the `ok` check as `LookupEnv` will return an empty string if the value isn't set.

Reply:
Thanks, I will make the update